package.json

```diff
@@ -15,31 +15,30 @@
     "typing:check": "tsc --noEmit"
   },
   "dependencies": {
-    "@googlemaps/js-api-loader": "^1.12.11",
-    "@radix-ui/react-separator": "^0.1.1",
-    "classnames": "^2.3.1",
-    "gray-matter": "^4.0.3",
-    "lightgallery": "^2.3.0",
-    "marked": "^4.0.5",
-    "next": "12.0.7",
-    "next-pwa": "^5.4.1",
-    "next-seo": "^4.28.1",
-    "pino": "^7.3.0",
-    "pino-pretty": "^7.2.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "@googlemaps/js-api-loader": "1.15.1",
+    "@radix-ui/react-separator": "1.0.1",
+    "classnames": "2.3.2",
+    "gray-matter": "4.0.3",
+    "lightgallery": "2.7.0",
+    "marked": "4.2.4",
+    "next": "13.0.6",
+    "next-pwa": "5.6.0",
+    "next-seo": "5.15.0",
+    "pino": "8.7.0",
+    "pino-pretty": "9.1.1",
+    "react": "18.2.0",
```

Comment:
so for simpler apps it's not that hard to migrate to react@18

Reply:
I had the same fellings :)